src/ui/src/composables/useBlueprintRun.ts

```diff
@@ -89,7 +117,17 @@ export function useBlueprintRun(
 		}
 	}
 
-	return { isRunning: readonly(isRunning), run };
+	async function stop() {
+		const activeRunId = wfbm.activeBlueprintRunId.value;
+		if(!activeRunId) return;
+		try {
+			await stopBlueprintRun(wf, activeRunId);
+		} catch (err) {
+			console.error("Failed to stop blueprint run:", err);
```

Comment:
Could you initialize `useLogger` and use it? It's just a proxy to `console` but it could be useful lat some point to centralize logs
```suggestion
			logger.error("Failed to stop blueprint run:", err);
```

Reply:
Just noticed this error is handled by wf tracking, I don't think we need this log... removed